myriad-scheduler/src/main/java/org/apache/myriad/scheduler/ServiceTaskFactoryImpl.java

```diff
@@ -232,7 +242,12 @@ public ExecutorInfo getExecutorInfoForSlave(FrameworkID frameworkId, Offer offer
         }
       }
     }
-    // this is actually an error condition - we did not have enough ports to use
+    //this is actually an error condition - we did not have enough ports to use
+    //TODO (hokiegeek2) does this need error handling then?
     return returnedPorts;
   }
-}
+  
+  private boolean isDefaultRole(Resource resource) {
+    return !resource.hasRole() || resource.getRole().equals("*");
+  }
```

Comment:
This is used elsewhere as well, worth adding to some static utility class?  Maybe not this PR, but something to think about.